replace-go.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
Non-blocking: Should the local build Dockerfile or utilities like this go in a subdirectory?